x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/aggregate/TopTests.java

```diff
@@ -53,6 +53,26 @@ public static Iterable<Object[]> parameters() {
                     .collect(Collectors.toCollection(() -> suppliers));
             }
         }
+        for (DataType valueType : DataType.values()) {
+            if (DataType.isString(valueType) == false) {
+                continue;
+            }
+            suppliers.add(new TestCaseSupplier(List.of(valueType), () -> {
```

Comment:
There's a `MultiRowTestCaseSupplier.stringCases()`, maybe use it with the other cases? It has a param for the expected DataType

Reply:
:+1: